articles/connections/passwordless/native-passwordless-universal.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Passwordless Authentication in Native Applications with Universal Login
+---
+# Passwordless Authentication in Native Applications with Universal Login
+
+[Universal Login](/hosted-pages/login) is the only recommended way to set up passwordless authentication in your native application. We recommend using the Auth0 login page for the best experience, security and the fullest array of features.
+
+Once you are done with this guide, the process should be that a user runs across a need to authenticate in your application and the Auth0 SDK redirects them to the login page on the web. At that point, Lock Passwordless will ask them for a phone number or email, whichever type you chose, which they will give, and the prompt will change to ask them for a code which they will receive by the designated metho. Once they enter the code, the transaction will finish and the user will be redirected to your app along with their credentials.
```

Comment:
typo: `metho`

Reply:
`the process should be that a user runs across a need to authenticate in your application and the Auth0 SDK redirects them to the login page on the web`

this needs to be simplified

Reply:
since it's a process I think it would be more clean and understandable as a series of steps (I would also draw a diagram showcasing the process and send it to #design-requests)